### DISCUSSION TITLE
Accidentally broke Win11 security updates

### DISCUSSION THREAD

**Discussion by natemcintosh** [undergroundwires/privacy.sexy#256] [Q&A]
I went through many options and checked them off. However, I seem to have accidentally broken the windows security update. I would like to fix this. Any suggestions?

![image](https://github.com/undergroundwires/privacy.sexy/assets/40805156/63b0b964-f0f2-4aeb-9bbc-783e096d7d06)

**Comment by undergroundwires** [comment]
Hi @natemcintosh , if you create an issue with [all necessary information](https://raw.githubusercontent.com/undergroundwires/privacy.sexy/0.12.3/.github/ISSUE_TEMPLATE/1-bug-report-scripts.md) to reproduce this with e.g. script you executed we can help you better. You can test running [revert script from this post](https://github.com/undergroundwires/privacy.sexy/issues/252#issuecomment-1705652055), it will be included in next patch release.

**Reply by natemcintosh** [reply]
Unfortunately I can't find the original script anymore. After running the revert script and restarting, the issue was not fixed. 

I ended up [using this](https://support.microsoft.com/en-us/windows/windows-update-troubleshooter-19bc41ca-ad72-ae67-af3c-89ce169755dd#WindowsVersion=Windows_11) documentation which seems to have resolved the issue for me.

**Comment by undergroundwires** [comment]
This was due to removal of Edge and it's fixed in a patch release long time ago
